lib/include/irritator/io.hpp

```diff
@@ -2113,7 +2114,9 @@ struct writer
 
     void write(const simulation& /*sim*/, const filter& dyn) noexcept
     {
-        os << "filter"<<dyn.default_value<<' '<<dyn.default_input_coeff <<'\n';
+        os << "filter " << dyn.default_lower_threshold << ' ' << dyn.default_upper_threshold
+           << ' ' << dyn.inValue
```

Comment:
Do not write `inValue`. This variable is a simulation variable.

Reply:
Thank you I will make the corrections